app/src/main/res/drawable/ic_database_off.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Why importing a drawable? This icon is available in [Google Fonts](https://fonts.google.com/icons?selected=Material+Symbols+Outlined:database_off:FILL@0;wght@400;GRAD@0;opsz@24&icon.query=database&icon.size=24&icon.color=%235f6368) and should be available as a vector drawable in Compose.

Reply:
I could not find it in `Icons.Default.*`. Not sure why ...

Reply:
And it's not in any of the others either ...
https://developer.android.com/reference/kotlin/androidx/compose/material/icons/Icons#summary

Reply:
Can you tell me how to add it? It would be much nicer indeed :+1:

Reply:
> https://developer.android.com/reference/kotlin/androidx/compose/material/icons/Icons#summary

It's strange, there are a lot of icons missing there. I'd say only the "regular" icons are there, none from extended.

Reply:
As a matter of fact, it looks like none of the "database" icons are available.

Reply:
I guess we could use [this](https://composeicons.com/icons/material-symbols/outlined/database_off). But it's still strange to me that they are not available...

Reply:
Nahr, I would leave it as is for now then. Maybe they add it later ... or not.

Reply:
If we want to show a notification we will need a drawable anyway, so I guess it works out.